### PR TITLE
Controller modifications for redhat

### LINKED ISSUE #106: Modify controller resource requirements
For install on RedHat Openshift, additional memory needed. Update defaults for all platforms.
Replaces https://github.com/iter8-tools/docs/issues/43 which was opened on the wrong repository.

### LINKED ISSUE #108: Make namespace configurable
Namespace where iter8 is installed is hardcoded to be `iter8` in a number of places. For Redhat, it seems like namespace `istio-system` might be a better choice.

Related to https://github.com/iter8-tools/iter8-controller/issues/107

### PR DESCRIPTION
**Pull request by kalantar**
Changes needed to work with Redhat.
Resolves #106 Resolves #108

### CODE REVIEW

File: install/helm/iter8-controller/templates/default/manager.yaml [hunk 1]
```diff
@@ -35,15 +36,17 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         name: manager
         env:
-          - name: POD_NAMESPACE
+          - name: SERVICE_NAMESPACE
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: WEBHOOK_PORT
```
Comment (kimikowang) [inline install/helm/iter8-controller/templates/default/manager.yaml:43]: I need further confirmation on the function of this port.

Comment (kimikowang) [reply install/helm/iter8-controller/templates/default/manager.yaml:43]: The controller manager should be configured to use this port as well. Just like how the namespace is set.

Comment (kalantar) [reply install/helm/iter8-controller/templates/default/manager.yaml:43]: @kimikowang I'm convinced that this port isn't causing any kind of issue for Redhat. I am inclined to undo my changes related to this. Any objection?

Comment (kimikowang) [reply install/helm/iter8-controller/templates/default/manager.yaml:43]: I am ok with it.

Comment (kalantar) [reply install/helm/iter8-controller/templates/default/manager.yaml]: @kimikowang I've backed out the changes and reverted things somewhat to minimize the changes.

**Pull request merged**